src/odemis/dataio/tiff.py

```diff
@@ -2911,6 +2912,7 @@ def _convert_zeiss_to_odemis_metadata(metadata: str) -> dict:
 
         # MD_POS should match image dimensions, e.g. only XY for 2D, XYZ for 3D, etc.
         md[model.MD_POS] = (x, y)
+        md[model.MD_ROTATION] = 0 # tmp: disable rotation metadata, as it is inconsistent with how odemis displays images
```

Comment:
No need to add this. The default is 0.
You can put a comment on MD_BEAM_SCAN_ROTATION to indicate we don't use MD_ROTATION because we don't know the reason of the scan rotation in the first place.